nemo/collections/tts/modules/submodules.py

```diff
@@ -44,8 +44,8 @@ def forward(self, input: torch.Tensor, mask_in: Tuple[int, int, int] = None):
         # if a mask is input, or tensor shape changed, update mask ratio
         if mask_in is not None or self.last_size != tuple(input.shape):
             # borisf: disabled update for inference
-            if self.training:
-                self.last_size = tuple(input.shape)
+            # if self.training:
+            self.last_size = tuple(input.shape)
```

Comment:
Why was this changed?

Reply:
When I brought this class in last cycle, it had caching disabled for inference. That was my quick fix for export and does not need to happen (caching for export is disabled on the other side).
Please remove the comment as well.

Reply:
updated with Boris's new patch